.github/actions/comment-pr-with-links/comments-with-url-links.js

```diff
@@ -1,51 +1,56 @@
-            let urls = [];
-            files.forEach(file => {
-                const splitted = file.split('/');
-                splitted.shift();
-                const pageName = splitted.pop();
-                const moduleName = splitted.shift();
-                let url = `${SITE_URL}/${COMPONENT_NAME}/${pr.base.ref}${moduleName === 'ROOT' ? '/' : `/${moduleName}/`}${pageName?.split('.').shift()}`;
-                urls.push(`- [ ] [${moduleName}/${pageName}](${url})`);
-            });
-            return urls.join('\n');
+    prepareUrlLinks: async function ({github, context}) {
+        let {FILES,DELETED, SITE_URL, COMPONENT_NAME} = process.env;
+        const {data: pr} = await github.rest.pulls.get({
+            owner: context.repo.owner,
+            pull_number: context.issue.number,
+            repo: context.repo.repo,
+        });
+        let result = {};
+        result.updated = prepareLinks({files: FILES.split(' '), siteUrl: SITE_URL, component: COMPONENT_NAME, branch: pr.base.ref});
+        result.deleted = prepareLinks({files: DELETED.split(' '), siteUrl: SITE_URL, component: COMPONENT_NAME, branch: pr.base.ref});
+        return result;
     },
-    createOrUpdateComments: async function ({github,context}){
-        let {LINKS, RENAMED_FILES, HAS_DELETED_FILES} = process.env;
-        const header='## :memo: Check the pages that have been modified\n\n';
-        let body = buildMessage(header,LINKS,HAS_DELETED_FILES === 'true' || RENAMED_FILES != '');
-        const {exists, id} = await githubUtils.isCommentExist({github,context,template: templateCommentId});
+    createOrUpdateComments: async function ({github, context}) {
+        let {LINKS, HAS_DELETED_FILES} = process.env;
+        const header = '## :memo: Check the pages that have been modified \n\n';
+        let links = JSON.parse(LINKS);
+        let body = buildMessage({header, links ,hasWarningMessage : (HAS_DELETED_FILES === 'true')});
+        const {exists, id} = await githubUtils.isCommentExist({github, context, template});
         // Delete oldest comment if another comments exist
-        if (exists && id){
-            await githubUtils.updateComment({github,context,comment_id: id, body});
+        if (exists && id) {
+            await githubUtils.updateComment({github, context, comment_id: id, body});
             return id;
         }
-        const comment = await githubUtils.createComment({github,context, body});
+        const comment = await githubUtils.createComment({github, context, body});
         return comment?.id;
     }
-
 };
 
-function buildMessage(header,links,hasWarningMessage){
+function buildMessage({header, links, hasWarningMessage}) {
     const preface =
         'In order to merge this pull request, you need to check your updates with the following url.\n\n';
 
-    const availableLinks = `### :mag: Page list: \n ${links}\n\n\n\n`;
-
+    const availableLinks = `### :mag: Page list: \n ${links.updated}\n\n\n`;
```

Comment:
I suggest we provide more context here, as we do for redirects/aliases.

```suggestion
    const availableLinks = `### :mag: Updated pages \n The following pages were updated, please ensure that the display is correct: \n ${links.updated}\n\n\n`;
```